migrations/858-create-limited-reviewers-group.sql

```diff
@@ -0,0 +1,3 @@
+-- Make Limited Reviewers group.
+INSERT INTO groups (name, rules, notes, created, modified) VALUES
+  ('Limited Reviewers', 'Addons:DelayedReviews', '', NOW(), NOW());
```

Comment:
could you add something to the notes column that explains what this group is intended for?